src/api.v2/helpers/access/postRegistrationHandler.js

```diff
@@ -0,0 +1,26 @@
+const parseSNSMessage = require('../../../utils/parse-sns-message');
```

Comment:
There are quite a few files whose names do not conform to the convention, especially for files under `/hooks`. I think rather than changing them here, would it be better if we have a PR specifically just to change the names of these functions? Although the changes are only renames, it will affect a lot of files,

Reply:
ok, good point. Let's do it in a separate pr